src/Safe7579Launchpad.sol

```diff
@@ -110,6 +125,28 @@ contract Safe7579Launchpad is IAccount, SafeStorage {
         });
     }
 
+    function addSafe7579(
```

Comment:
additional function that allows existing safe accounts to get safe7579 with a single transaction